src/DevLicenseCore.sol

```diff
@@ -14,46 +13,46 @@ import {IDimoToken} from "./interface/IDimoToken.sol";
 /**
  * @title Developer License Core
  * @dev Implements the core functionalities for managing developer licenses within the DIMO ecosystem.
+ * @dev To facilitate potential upgrades, this agreement employs the Namespaced Storage Layout (https://eips.ethereum.org/EIPS/eip-7201)
  * @notice This contract manages the creation, administration, and validation of developer licenses,
  *         integrating with DIMO's token and credit systems.
  */
-contract DevLicenseCore is IDevLicenseDimo, AccessControl {
-    error AliasAlreadyInUse(bytes32 licenseAlias);
-    /*//////////////////////////////////////////////////////////////
-                             Access Controls
-    //////////////////////////////////////////////////////////////*/
+contract DevLicenseCore is Initializable, AccessControlUpgradeable, IDevLicenseDimo {
+    /// @custom:storage-location erc7201:DIMOdevLicense.storage.DevLicenseCore
+    struct DevLicenseCoreStorage {
+        IDimoToken _dimoToken;
```

Comment:
I'm curious—why give all of these a `_` prefix?

Reply:
It was a pattern that was already in many parts of the code. I don't think I have a strong reason to keep or remove it.